src/scss/dp/overrides/components/_header.scss

```diff
@@ -0,0 +1,477 @@
+// _header.scss
```

Comment:
There are some nav links in the header in mobile view where the font and/or text-decoration colour needs to be overridden from the ONS DS' default:

![image](https://user-images.githubusercontent.com/23668262/140306998-b62abe0d-36cc-4b34-b33a-b52543157654.png)

![image](https://user-images.githubusercontent.com/23668262/140307083-8e56fd1c-aea7-4698-bc74-66cdb78f9791.png)